From 375b572fa2c8343294de06d6875ddb6db993fa8c Mon Sep 17 00:00:00 2001
From: tedyu <yuzhihong@gmail.com>
Date: Thu, 21 May 2015 13:58:33 -0700
Subject: [PATCH 185/226] HBASE-13741 Disable TestRegionObserverInterface#testRecovery and testLegacyRecovery (Stephen Jiang)

(cherry picked from commit 86e2c3b72fb5d96bfab9cde90292fc1f650b6900)

Reason: Test instable
Author: Stephen Jiang
Ref: CDH-24489
---
 .../coprocessor/TestRegionObserverInterface.java   |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/hbase-server/src/test/java/org/apache/hadoop/hbase/coprocessor/TestRegionObserverInterface.java b/hbase-server/src/test/java/org/apache/hadoop/hbase/coprocessor/TestRegionObserverInterface.java
index 031e173..28a9a8e 100644
--- a/hbase-server/src/test/java/org/apache/hadoop/hbase/coprocessor/TestRegionObserverInterface.java
+++ b/hbase-server/src/test/java/org/apache/hadoop/hbase/coprocessor/TestRegionObserverInterface.java
@@ -77,6 +77,7 @@ import org.apache.hadoop.hbase.util.JVMClusterUtil;
 import org.apache.hadoop.hbase.util.Threads;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
@@ -586,6 +587,7 @@ public class TestRegionObserverInterface {
     }
   }
 
+  @Ignore // TODO: HBASE-13391 to fix flaky test
   @Test (timeout=300000)
   public void testRecovery() throws Exception {
     LOG.info(TestRegionObserverInterface.class.getName() +".testRecovery");
@@ -636,6 +638,7 @@ public class TestRegionObserverInterface {
     }
   }
 
+  @Ignore // TODO: HBASE-13391 to fix flaky test
   @Test (timeout=300000)
   public void testLegacyRecovery() throws Exception {
     LOG.info(TestRegionObserverInterface.class.getName() +".testLegacyRecovery");
@@ -771,4 +774,4 @@ public class TestRegionObserverInterface {
       writer.close();
     }
   }
-}
\ No newline at end of file
+}
-- 
1.7.0.4

